packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -66,31 +75,48 @@ const propTypes = {
    * Toggle to show notification-dialog or not.
    */
   isOpen: PropTypes.bool.isRequired,
+  /**
+   * Reverses the order of notification action buttons
+   */
+  isReversed: PropTypes.bool,
```

Comment:
I'd rather not have a default value for the button order and require consumers to decide which button order they want. This should be the breaking change for this update.

Reply:
Replaced with `buttonOrder` prop that is required https://github.com/cerner/terra-framework/pull/899/commits/67555271fd37cfa811ddb2cf31151f65149929b1#diff-cb62dc7ce0197bc569a9952450bb9fe5R98